src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs

```diff
@@ -277,6 +298,37 @@ public DisplayMetricsCache(DisplayMetrics displayMetric)
 			public float Xdpi { get; }
 
 			public float Ydpi { get; }
+
+			private DisplayMetricsDensity ConvertIntToDensityEnum(int DPI)
+			{
+				if (DPI == 120) return DisplayMetricsDensity.Low;
+				if (DPI == 160) return DisplayMetricsDensity.Medium;
+				if (DPI == 240) return DisplayMetricsDensity.High;
+				if (DPI == 320) return DisplayMetricsDensity.Xhigh;
+				if (DPI == 480) return DisplayMetricsDensity.Xxhigh;
+				if (DPI == 640) return DisplayMetricsDensity.Xxxhigh;
+
+				if (DPI == 213) return DisplayMetricsDensity.Tv;
+
+				if (DPI < 120) return DisplayMetricsDensity.Low;
+				if (DPI < 160) return DisplayMetricsDensity.D140;
+				if (DPI < 180) return DisplayMetricsDensity.D180;
+				if (DPI < 200) return DisplayMetricsDensity.D200;
+				if (DPI < 220) return DisplayMetricsDensity.D220;
+				if (DPI < 260) return DisplayMetricsDensity.D260;
+				if (DPI < 280) return DisplayMetricsDensity.D280;
+				if (DPI < 300) return DisplayMetricsDensity.D300;
+				if (DPI < 340) return DisplayMetricsDensity.D340;
+				if (DPI < 360) return DisplayMetricsDensity.D360;
+				if (DPI < 400) return DisplayMetricsDensity.D400;
+				if (DPI < 420) return DisplayMetricsDensity.D420;
+				if (DPI < 440) return DisplayMetricsDensity.D440;
+				if (DPI < 450) return DisplayMetricsDensity.D450;
+				if (DPI < 560) return DisplayMetricsDensity.D560;
```

Comment:
You should now be able to use a switch expression statement.

Reply:
Convert it a follows:
```csharp
    return DPI switch
    {
        120 => DisplayMetricsDensity.Low,
        160 => DisplayMetricsDensity.Medium,
        240 => DisplayMetricsDensity.High,
        320 => DisplayMetricsDensity.Xhigh,
        480 => DisplayMetricsDensity.Xxhigh,
        640 => DisplayMetricsDensity.Xxxhigh,
        213 => DisplayMetricsDensity.Tv,
        < 120 => DisplayMetricsDensity.Low,
        < 160 => DisplayMetricsDensity.D140,
        < 180 => DisplayMetricsDensity.D180,
        < 200 => DisplayMetricsDensity.D200,
        < 220 => DisplayMetricsDensity.D220,
        < 260 => DisplayMetricsDensity.D220,
        < 280 => DisplayMetricsDensity.D280,
        ....
        _ => DisplayMetricsDensity.D600
    };
```

If you use VS 2022, intellisense automatically suggests you to convert the ifs to a switch expression,